ginga/rv/plugins/WCSAxes.py

```diff
@@ -0,0 +1,305 @@
+        except Exception:
+            pass
+
+        # dc.WCSAxes will pick up image under the hood.
+        # UNTIL HERE - what about other params to adjust?
+        obj = self.dc.WCSAxes(
+            linewidth=self.linewidth, linestyle=self.linestyle,
+            color=self.linecolor, alpha=self.alpha, fontsize=self.fontsize)
+        self.overlaytag = self.canvas.add(obj)
+        self.canvas.redraw(whence=3)
+
+    def set_linecolor_cb(self, w, index):
+        self.linecolor = self.colornames[index]
+        self.redo()
+        return True
+
+    def set_alpha(self):
+        try:
+            a = float(self.w.alpha.get_text())
+        except ValueError:
+            self.w.alpha.set_text(str(self.alpha))
+        else:
+            self.alpha = a
+            self.redo()
+        return True
+
+    def set_linestyle_cb(self, w, index):
+        self.linestyle = self.linestyles[index]
+        self.redo()
+        return True
+
+    def set_num_ra(self):
+        try:
+            n = int(self.w.num_ra.get_text())
+        except ValueError:
+            self.w.num_ra.set_text(str(self.num_ra))
+        else:
+            self.num_ra = n
+            self.redo()
+        return True
+
+    def set_num_dec(self):
+        try:
+            n = int(self.w.num_dec.get_text())
+        except ValueError:
+            self.w.num_dec.set_text(str(self.num_dec))
+        else:
+            self.num_dec = n
+            self.redo()
+        return True
+
+    def toggle_label_cb(self, w, val):
+        self.show_label = val
+        self.redo()
+
+    def set_fontsize(self):
+        try:
+            val = float(self.w.font_size.get_text())
+        except ValueError:
+            self.w.font_size.set_text(str(self.fontsize))
+        else:
+            self.fontsize = val
+            self.redo()
+        return True
+
+    def set_ra_angle(self):
+        try:
+            a = float(self.w.ra_angle.get_text())
+        except ValueError:
+            self.ra_angle = None
+            self.w.ra_angle.set_text(str(self.ra_angle))
+        else:
+            self.ra_angle = a
+        self.redo()
+        return True
+
+    def set_dec_angle(self):
+        try:
+            a = float(self.w.dec_angle.get_text())
+        except ValueError:
+            self.dec_angle = None
+            self.w.dec_angle.set_text(str(self.dec_angle))
+        else:
+            self.dec_angle = a
+        self.redo()
+        return True
+
+    def close(self):
+        self.fv.stop_local_plugin(self.chname, str(self))
+        return True
+
+    def start(self):
+        # insert canvas, if not already
+        p_canvas = self.fitsimage.get_canvas()
+        try:
+            p_canvas.get_object_by_tag(self.layertag)
+        except KeyError:
+            # Add ruler layer
+            p_canvas.add(self.canvas, tag=self.layertag)
+
+        self.resume()
+
+    def pause(self):
+        self.canvas.ui_set_active(False)
```

Comment:
I don't think there is any drawing or UI on the canvas, so probably no need for `pause()` and `resume()` methods.